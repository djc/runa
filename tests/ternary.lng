def __main__(name: str, args: list[str]) -> int:
	
	print(str(1 if 0 else 2))
	print(str(2 if 111 else 1))
	
	print('true' if 111 else 'false')
	print('true' if 0 else 'false')
	
	print(str(1 if 'a' else 0))
	print(str(1 if '' else 0))
	
	print(str(1 if not '' else 0))
	print('a' and 'b')
	print('' and 'a')
	print('a' and '')
	print('a' or 'b')
	print('' or 'a')
	print('a' or '')
