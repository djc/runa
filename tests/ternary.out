2
2
true
false
1
0
0
b


a
a
a
